manoseimas/mps_v2/views/json.py

```diff
@@ -140,6 +146,26 @@ def suggesters_json(request):
                          'subtab_counts': subtab_counts()})
 
 
+def _mangle(text):
+    text = text.lstrip()
+    text = re.sub(u'^(PAGRINDINIO )?(KOMITETO )?(IŠVADA )?', u'', text)
+    text = text.lstrip()
+    return text
+
+def _resolution_dict(resolution):
+    return {
+        'title': _mangle(resolution.title),
+        'url': resolution.source,
+    }
+
+def resolutions_json(request, suggester_slug):
+    """Get resolutions for a given suggester."""
+    resolutions_qs = CommitteeResolution.objects.filter(
+        suggestion__submitter__slug=suggester_slug).distinct()
```

Comment:
I got multiple resolutions with the same title and the same URL. I have not checked the key. 

Hey, let's look up the QS. Also, let'l look up the SQL.